cfme/tests/automate/custom_button/test_service_vm_custom_button.py

```diff
@@ -228,3 +228,54 @@ def test_custom_button_with_dynamic_dialog_vm(
             submit = "submit" if context is ViaUI else "submit_request"
             getattr(view, submit).click()
             log.wait_for_log_validation()
+
+
+@pytest.mark.meta(automates=[1427430, 1450473])
+def test_custom_button_automate_service_vm(request, appliance, service_vm, button_group):
+    """ Test custom button execution on SSUI vm resource detail page
+
+    Polarion:
+        assignee: ndhandre
+        initialEstimate: 1/2h
+        caseposneg: positive
+        testtype: functional
+        startsin: 5.9
+        casecomponent: CustomButton
+        tags: custom_button
+
+    Bugzilla:
+        1427430
+        1450473
+        1454910
+    """
+
+    service, _ = service_vm
+    with appliance.context.use(ViaUI):
+        button = button_group.buttons.create(
+            text=fauxfactory.gen_alphanumeric(),
+            hover=fauxfactory.gen_alphanumeric(),
+            system="Request",
+            request="InspectMe",
+        )
+        request.addfinalizer(button.delete_if_exists)
+
+    # Check for UI and SSUI destinations.
+    for context in [ViaUI, ViaSSUI]:
```

Comment:
@digitronik thoughts on parameterizing this test case against `context`?

Reply:
right, we have a tradition to parametrize a test with ViaUI` and `ViaSSUI` with the help of `santaku`. but I'm thinking to make it more simple and inclusive.

1. In this case, it's possible also but no need of this as we are checking a single button on different `destinations`.
2. If we did, test time will dobule; which is not good when the test needs more `setup steps` or `timing`. `service_vm` fixture is in `functional` scope. I won't want to spin VM just for button execution on different destinations.

Reply:
That makes sense, thanks for the explanation!